### ISSUE TITLE
Reality not available for the latest build

### ISSUE THREAD

**Issue by narakai** [aaaamirabbas/nekoray-macos#3]
Hi,
Thanks for the last build, it works on VMess but seems Reality not  included in this build.
Logs for this issue:
create service: parse outbound[0]: uTLS, which is required by reality client is not included in this build, rebuild with -tags with_utls

**Comment by aaaamirabbas** [comment]
Hi, I don't have information about something called `Reality Client‍`
Please explain more or link to me about this.

Also, consider this repository is builder based of the main source, so the main Nekoray developers need to add new options to the software or fix the bug.

To find out if you find the problem in the new buildings, please install and test version 2.12 from the main repository.

**Comment by narakai** [comment]
Hi, it is a new protocol which has been added in X-ray core v1.8.0, and has been included since version 2.22, you can check at: https://github.com/MatsuriDayo/nekoray/issues/512
https://github.com/MatsuriDayo/nekoray/releases

By the way, I've checked version 2.25 for Windows, the protocol works fine on it.

**Comment by aaaamirabbas** [comment]
**I found the problem**, I will upload a new version in the next few hours.
Please test and report the new version.

**Comment by narakai** [comment]
New version works perfect, thanks for the great work.
I tried to build it myself using this script, but always fail because qt5 can not be found on Mac, even if I have installed it using brew. Can you give some advice on how to solve this, thanks.

**Comment by Stevemoretz** [comment]
> New version works perfect, thanks for the great work. I tried to build it myself using this script, but always fail because qt5 can not be found on Mac, even if I have installed it using brew. Can you give some advice on how to solve this, thanks.

Can you confirm that reality works for you on the latest version? https://github.com/Stevemoretz/nekoray-macos/releases/tag/2.26

**Comment by narakai** [comment]
@Stevemoretz reality works on this version.

**Comment by narakai** [comment]
I can't tell what is wrong from the logs, do you get your setting right, should be something like: 

<img width="846" alt="1" src="https://user-images.githubusercontent.com/6311647/232828295-f88e4d1f-ff73-48a7-aed7-63519c4cac7e.png">

**Comment by Stevemoretz** [comment]
@narakai Thanks for the reply!

That's very strange, mine doesn't look like that at all, I don't have the Flow under VLESS, and I have no RealityPbk* under TLS Camouflage Settings!

<img width="820" alt="Screen Shot 1402-01-29 at 19 06 45" src="https://user-images.githubusercontent.com/27680142/232828979-3d3b98de-f1c6-4a36-9b90-8d0e7368baf5.png">

But I am using the latest version 2.26
<img width="186" alt="Screen Shot 1402-01-29 at 19 08 21" src="https://user-images.githubusercontent.com/27680142/232829357-7993188a-05d6-469c-8bc2-7ea5bb9b3271.png">

**Comment by narakai** [comment]
@Stevemoretz I tried but your config doesn't work for me.

**Comment by narakai** [comment]
@Stevemoretz Checked again, now it works, you can remove your real ip.

**Comment by Stevemoretz** [comment]
> @Stevemoretz Checked again, now it works, you can remove your real ip.

@narakai Thanks! So you just import from clipboard and the same config works for you but for me it doesn't even bring the same settings as you!

@aaaamirabbas Looks like a bug! Could you take a look at that?

**Comment by Stevemoretz** [comment]
> @Stevemoretz Checked again, now it works, you can remove your real ip.

@narakai I remembered a note about using NEKO_BOX, to activate reality 
https://github.com/MatsuriDayo/nekoray/commit/d6de7e4affd46efec4885d10fbe6a91529631c40#diff-ca03d3c0fd749844b0f67d87779a4c35c3ce311f3c782e12fdd202a6096b6045R42

Are you using neko box? What is that?

**Comment by narakai** [comment]
Yes, Reality is Neko Box only, according to the releases log : https://github.com/MatsuriDayo/nekoray/releases.

**Comment by narakai** [comment]
Try: Basic Settings - Core - Switch Core - sing-box

**Comment by Stevemoretz** [comment]
> Try: Basic Settings - Core - Switch Core - sing-box

That was it 👍  the settings came for me too and I am connected to it right now! Thank you so much "sing-box" = "neko-box" lol.
